lib/attack_test.go

```diff
@@ -358,3 +359,21 @@ func TestClient(t *testing.T) {
 		t.Errorf("Expected timeout error")
 	}
 }
+
+func TestSeqHeader(t *testing.T) {
+	t.Parallel()
+
+	var got http.Header
+
+	atk := NewAttacker()
+	for i := 0; i < 5; i++ {
```

Comment:
Here I'm looping to ensure that we are not only testing `res.Seq == 0`. I can remove it if you prefer.

Reply:
Sounds good, just add that as code comment.